src/current/v25.2/architecture/transaction-layer.md

```diff
@@ -432,6 +432,36 @@ Despite their logical equivalence, the transaction coordinator now works as quic
 
 Additionally, when other transactions encounter a transaction in `STAGING` state, they check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating that staging transaction's record. If the coordinator is still heartbeating the record, the other transactions will wait, on the theory that letting the coordinator update the transaction record with the result of the attempt to commit will be faster than going through the transaction status recovery process. This means that in practice, the transaction status recovery process is only used if the transaction coordinator dies due to an untimely crash.
 
+### Buffered writes
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v25.2" %} Buffered writes enhance transaction throughput and reduce operational cost by minimizing the number of round trips between the gateway node and other nodes during write operations.
```

Comment:
One thing I forgot to mention is that it's not clear whether buffered writes are something you enable, or the new way that CRDB works. It's not until most of the way through this section that you see it's something you enable. It might be worth clarifying that briefly here. "Buffered writes are a new, optional feature that temporarily stores..." or something like that.

Reply:
Alternatively, move lines 451-459 to line 443.